hudi-client/hudi-client-common/pom.xml

```diff
@@ -179,7 +179,11 @@
         </exclusion>
       </exclusions>
     </dependency>
-
+    <dependency>
+      <groupId>org.mockito</groupId>
+      <artifactId>mockito-inline</artifactId>
```

Comment:
Why is this new dependency needed? For mocking static methods? If so, can we not make it just private and visible for testing, and mock metaclient if needed.

Reply:
yeah, this is needed for mock static method. I will make the dependency more properly.

Reply:
Removed.

Reply:
I found that I cannot remove this, since they do needed.